praetorian_cli/sdk/chariot.py

```
@@ -183,8 +183,11 @@ def update(self, type: str, body: dict, params: dict = {}) -> dict:
     def upsert(self, type: str, body: dict, params: dict = {}) -> dict:
         return self.put(type, body, params)
 
-    def link_account(self, username: str, value: str = '', config: dict = {}) -> dict:
-        resp = self.chariot_request('POST', self.url(f'/account/{username}'), json=dict(config=config, value=value))
+    def link_account(self, username: str, role: str = '', value: str = '', config: dict = {}) -> dict:
```

Ruff (0.15.6)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)